.github/workflows/build-images-action.yml

```diff
@@ -6,20 +6,38 @@ permissions:
 on:
   push:
     branches:
-    - 'main'
-    - 'release-*'
+    - "main"
+    - "release-*"
     tags:
-    - 'v*'
+    - "v*"
+  workflow_run:
+    workflows:
+    - "Create Release"
+    types:
+    - completed
 
 jobs:
   build_ipam:
-    name: Build IPAM container image
-    if: github.repository == 'metal3-io/ip-address-manager'
-    uses: metal3-io/project-infra/.github/workflows/container-image-build.yml@main
-    with:
-      image-name: 'ip-address-manager'
-      pushImage: true
-    secrets:
-      QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
-      QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
-      SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set ref
+      id: set_ref
+      run: |
+        # Check if release_tag exists, if so, set ref to "refs/heads/${{ release_tag }}", otherwise use github.ref
+        if [[ -n "${{ needs.create-release.outputs.release_tag }}" ]]; then
```

Comment:
What sets this?

Reply:
It's inside the "Create Release" workflow.

Reply:
Yes, but how does it end up being set here? What populates the `needs.` so it can be used here?

Reply:
It's this line: https://github.com/metal3-io/ip-address-manager/blob/main/.github/workflows/release.yaml#L19